SwiftServer/Sources/Server/ConfigureRoutes.swift

```diff
@@ -0,0 +1,48 @@
+import HTTPResponse
+
+public protocol Route {
+    var url: String { get }
+    func handleRequest() -> HTTPResponse
+}
+
+struct RootRoute: Route {
+    var url: String
+
+    func handleRequest() -> HTTPResponse {
+        return HTTPResponse()
+                .setVersion(version: 1.1)
+                .setResponseCode(responseCode: ResponseCodes.OK)
+                .setMessage(message: "Hello!")
+    }
+}
+
+struct methodOptions: Route {
+    var url: String
+
+    func handleRequest() -> HTTPResponse {
+        return   HTTPResponse()
+                .setVersion(version: 1.1)
+                .setResponseCode(responseCode: ResponseCodes.OK)
+                .addHeader(key: "Allow", value: "GET,HEAD,POST,OPTIONS,PUT")
```

Comment:
Not completely sure how I'm going to set this up, but at least tried to think about and move away from hard-coding 2074ca9.